Makefile

```diff
@@ -1,10 +1,13 @@
+FILE=schemas_version.txt
+VARIABLE=`cat $(FILE)`
```

Comment:
FILE, VARIABLE? Come on @petechd, you're better than this. 😄

Reply:
I'm not familiar with this syntax but I'll give it a try😅

Reply:
Lets hide this shameful code🤦‍♂️